RecoTracker/LSTCore/interface/alpaka/LSTESData.h

```diff
@@ -1,11 +1,9 @@
 #ifndef LSTESData_H
 #define LSTESData_H
 
-#ifdef LST_IS_CMSSW_PACKAGE
```

Comment:
this is from #47 and also seems to be only partial.

I'm tempted to suggest we merge https://github.com/SegmentLinking/cmssw/compare/CMSSW_14_1_0_pre3_LST_X_LSTCore_realfiles_batch3...remove_LST_IS_CMSSW_PACKAGE 
even though it includes the typo `+` -> `*` fix as well as the cleaning flags.

It may also work OK to cherry-pick dc13ac526b32b4c3ad1b391fb7b97511ce7802ea to be incremental wrt batch3

Reply:
Yeah, I think it would be good to merge (or cherry pick).